src/stories/pages/getting-started/GettingStarted.js

```diff
@@ -104,16 +104,15 @@ export class GettingStartedPage extends Page {
         <p class="overview_lottie_header">Relax</p>
         <div id="lottie3" class="overview-lottie" style="width: 116"></div>
         <p style="margin: 0">
-          The NWB Assistant comes with intuitive user interfaces and automation that streamline the
-          process for you
+          Use our intuitive user interface and automations to streamline your process
```

Comment:
```suggestion
          Use our intuitive interface and automations to streamline your process
```

To prevent 'use' from occurring twice in a 4 word span